rendering/renderlink.go

```diff
@@ -37,6 +37,9 @@ func (r RenderLink) CreateLink(gm *game.Game) (*url.URL, error) {
 	q.Add("from", from)
 	q.Add("to", to)
 	q.Add("check", check)
+	if gm.Turn() == "Black" {
```

Comment:
@ZurabWeb I think there is a constant for this value in the chess library we are using.